### DISCUSSION TITLE
Physlight - Camera Model

### DISCUSSION THREAD

**Discussion by xeggbee** [colour-science/colour#1101] [Q&A] [answered]
Virtual Camera model

Reviewed the "PhysLight: An End-to-End Pipeline for Scene-Referred Lightin" ppt and the tutorial.

https://github.com/wetadigital/physlight

So based in the spectral sensitivities of a camera and the illuminant, we can build a virtual colorchecker, and if i understand correctly, even the colorchecker not present in the image in the moment of acquisition we can "estimate"(based on solve_matrix function) how the colorchecker would be, and consequently make a color corretion based on virtual colorchecker. 
I`d like to confirm if understood correctly the PhysLight method.

regards!

**Comment by thomasmansencal** [comment, marked as answer]
Yes this is correct!

Here is a thread and a notebook that uses Hyper-Spectral Images and simulates some of those aspects: https://community.acescentral.com/t/input-device-transform-physical-camera-spectral-simulation/3647
